frontend/src/pages/Page/Page.tsx

```diff
@@ -1,4 +1,5 @@
 import React from 'react'
+import { isMac } from '../../services/Platform'
```

Comment:
Maybe use `Environment` instead?

Reply:
Maybe you mean environment from the backend? I don't have that available in the front end.